sync-script/README.md

```diff
@@ -0,0 +1,26 @@
+## Script to adapt the documents for the official docs page

```

Comment:
Can we please have these files under Specifications/sync-script instead?

Reply:
Yes, I'll change that